ArduCopter/mode_guided.cpp

```diff
@@ -867,6 +879,8 @@ void ModeGuided::angle_control_run()
     // call attitude controller
     if (guided_angle_state.use_yaw_rate) {
         attitude_control->input_euler_angle_roll_pitch_euler_rate_yaw(roll_in, pitch_in, yaw_rate_in);
+    } else if (guided_angle_state.use_roll_pitch_rates) {
+        attitude_control->input_rate_bf_roll_pitch_yaw(guided_angle_state.roll_rate_cds, guided_angle_state.pitch_rate_cds, guided_angle_state.yaw_rate_cds);
```

Comment:
In order to timeout the rates you need to have something like this with the appropriate setting of the rates to 0.0 above in the timeout section around L845.

```suggestion
        attitude_control->input_rate_bf_roll_pitch_yaw(roll_rate_in, pitch_rate_in, yaw_rate_in);
```

Reply:
Add roll and pitch rate timeouts here 8956875.